src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -495,6 +652,38 @@ def __init__(self, hits):
         self.apiResponse = ApiResponse(**class_entries)
 
 
+class ProjectKeywordSearchResponse(KeywordSearchResponse):
+    def make_projects(self, entry):
```

Comment:
This duplicates too much functionality from the overridden method. Take a look at the Template Method design pattern. I think that could be used to solve this problem.